pkg/tecdsa/signing/message.go

```diff
@@ -170,4 +170,24 @@ func (trsm *tssRoundSevenMessage) SenderID() group.MemberIndex {
 // marshaling purposes.
 func (trfm *tssRoundSevenMessage) Type() string {
 	return messageTypePrefix + "tss_round_seven_message"
+}
+
+// tssRoundEightMessage is a message payload that carries the sender's
+// TSS round eight components.
+type tssRoundEightMessage struct {
+	senderID group.MemberIndex
+
+	payload   []byte
```

Comment:
Let's make it `broadcastPayload` to keep it aligned across all types of messages.

Reply:
Addressed by https://github.com/keep-network/keep-core/pull/3271